src/legacy/core_plugins/console/public/quarantined/src/input.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// @ts-ignore
+import Autocomplete from './autocomplete';
+
+import { LegacyEditor } from '../../../np_ready/public/application/models';
+
+// @ts-ignore
+import SenseEditor from './sense_editor/editor';
+
+let input: any;
+export function initializeInput($el: JQuery<HTMLElement>, $actionsEl: JQuery<HTMLElement>) {
```

Comment:
Could we rename `initialzeInput` --> `initializeEditor`? And also `let input` to `let editor`. I might have worked too long with forms lately but when I see `input` my mental model is a form input. 😊